native/src/components/NavigationTiles.tsx

```diff
@@ -92,8 +88,7 @@ const NavigationTilesWithScrollableView = ({ tiles, theme }: PropsType): ReactEl
           name='keyboard-arrow-right'
           isLeftAnchor={false}
           scrollViewRef={scrollViewRef.current}
-          updatePosition={updatePosition}
-          disabled={currentPosition >= navigationItemWidth}
+          disabled={currentPosition >= lastItemAutoScroll * navigationItemWidth}
```

Comment:
Instead you could probably just use `onMomentumScrollEnd` to listen for the current position of the scroll view, i.e. remove `onScroll` and pass `handleScroll` to `onMomentumScrollEnd`
``` ts
     /**
     * Fires when scroll view has finished moving
     */
    onMomentumScrollEnd?: ((event: NativeSyntheticEvent<NativeScrollEvent>) => void) | undefined;
```

Reply:
This does not work for clicking the arrows, as this is no momentum scrolling. I added it nevertheless, as disabling is working smoother with it.